program/rust/src/accounts/price.rs

```diff
@@ -340,3 +344,23 @@ impl PythOracleSerialize for TwapMessage {
         bytes.to_vec()
     }
 }
+
+impl PythOracleSerialize for PublisherStakeCapsMessage {
+    fn to_bytes(self) -> Vec<u8> {
```

Comment:
I think having some sort versioning here can be helpful for future changes to this message.

Reply:
Won't do it since the other messages don't have it